mdoc/mdoc.csproj

```diff
@@ -37,21 +37,21 @@
     <Reference Include="System.Xml.Linq" />
     <Reference Include="ICSharpCode.SharpZipLib" />
     <Reference Include="System.Xml" />
-    <Reference Include="monodoc">
-      <HintPath>..\..\class\lib\net_4_x\monodoc.dll</HintPath>
+    <Reference Include="Mono.Cecil.Rocks">
+      <HintPath>..\external\Mono.Cecil\bin\net_4_0_Release\Mono.Cecil.Rocks.dll</HintPath>
```

Comment:
With a top level `bin`, `%(Reference.HintPath)` needs to change accordingly:

    <HintPath>..\bin\$(Configuration)\Mono.Cecil.Rocks.dll</HintPath>